### PR TITLE
Made corrections in redirect to Api-Hub and a fix in comp>index.ts(added apiHub home Fixes too)

### PR DESCRIPTION
**Pull request by Vinyl-Davyl**

### CODE REVIEW

File: src/components/index.ts [hunk 1]
```diff
@@ -8,5 +8,4 @@ export { default as SplitView } from "./SplitView";
 export { default as TabPanel } from "./TabPanel";
 export { default as UserProSideBar } from "./UserProSideBar";
 export { default as Widget } from "./Widget";
-export { default as DataTable } from "./DataTable"
```
Comment (taiwoAkindele) [inline src/components/index.ts:11]: why did you delete this?

Comment (Vinyl-Davyl) [reply src/components/index.ts:11]: It was repeated in line 2, Typescript sorted it out before running code. Check on your end in components>index.ts

**Review by pablo-clueless — APPROVED** [review]

File: src/components/index.ts (hunk 1 above)
Comment (taiwoAkindele) [reply src/components/index.ts:11]: Not there. okay

**Review by taiwoAkindele — APPROVED** [review]

File: src/components/index.ts (hunk 1 above)
Comment (Vinyl-Davyl) [reply src/components/index.ts:11]: Okay

**Review by PhilemonBrain — APPROVED** [review]
